### PR TITLE
Update Missing translations

### PR DESCRIPTION
**Pull request by thelinuxfr**
The beginning of my modest contribution, I am not high level PHP. Tell me if it's good? I try to help as I can ...

### CODE REVIEW

File: src/www/diag_authentication.php [hunk 1]
```diff
@@ -102,7 +102,7 @@ function getUserGroups($username, $authcfg)
                 <div class="content-box">
 
 					<header class="content-box-head container-fluid">
-				   <h3>Test a server</h3>
+				   <h3><?= gettext('Test a server') ?></h3>
```
Comment (fichtner) [inline src/www/diag_authentication.php]: this one is perfect :)

File: src/www/diag_dns.php [hunk 2]
```diff
@@ -77,7 +77,7 @@
 			$newalias['name'] = $aliasname;
 			$newalias['type'] = "network";
 			$newalias['address'] = $addresses;
-			$newalias['descr'] = "Created from Diagnostics-> DNS Lookup";
+			$newalias['descr'] = gettext("Created from Diagnostics-> DNS Lookup");
```
Comment (fichtner) [inline src/www/diag_dns.php]: Please always use single quotes for gettext strings (like above) -- they cannot be dynamic anyway and it's a little faster this way.

File: src/www/diag_dns.php [hunk 3]
```diff
@@ -207,14 +207,14 @@ function display_host_results ($address,$hostname,$dns_speeds) {
 							          <td>
 								      <? if($found > 0): ?>
 											  <?PHP	if($alias_exists): ?>
-												An alias already exists for the hostname <?= htmlspecialchars($host) ?>. <br />
+												<?=gettext("An alias already exists for the hostname"); ?> <?= htmlspecialchars($host) ?>. <br />
```
Comment (fichtner) [inline src/www/diag_dns.php]: Always wrap the whole sentence for context using sprintf to bring dynamic elements into the translation string:

    <?= sprintf(gettext(-An alias already exists for the hostname %s.'), htmlspecialchars($host)) ?>

(same below)

**Comment by fichtner** [comment]
Hi Frédéric,

Looking good. Some comments inline and then we'll clean up the pull request branch before merging. Thanks for this. :)


Cheers,
Franco

**Comment by thelinuxfr** [comment]
Hi,

OK what normally is corrected. No need to add strings in the PO file?

Thx

File: src/www/diag_dns.php [hunk 4]
```diff
@@ -207,14 +207,14 @@ function display_host_results ($address,$hostname,$dns_speeds) {
...
-												An alias already exists for the hostname <?= htmlspecialchars($host) ?>. <br />
+												<?= sprintf(gettext('An alias already exists for the hostname %s.'), htmlspecialchars($host)) ?>. <br />
 												<input type="hidden" name="override" value="true"/>
 												<input type="submit" class="btn btn-default" name="create_alias" value="Overwrite Alias"/>
 											  <?php else: ?>
 											  <?php if(!$createdalias): ?>
 												<input type="submit" class="btn btn-default" name="create_alias" value="Create Alias"/>
 											  <?php else: ?>
-											   Alias created with name <?= htmlspecialchars($newalias['name']) ?>
+											   <?= sprintf(gettext('Alias created with name'), htmlspecialchars($newalias['name'])) ?>
```
Comment (fichtner) [inline src/www/diag_dns.php]: %s is missing here now

**Comment by fichtner** [comment]
Strings for pot/po files are picked up by another workflow later (they can be automatically extracted from gettext calls) :)

**Comment by fichtner** [comment]
Hey, I saw you kept working on it just now but I don't get any notifications from GitHub for this. Let me know when you are ready for feedback via comment, alright? :)

**Comment by thelinuxfr** [comment]
hi, i'm ready ;)

**Comment by fichtner** [comment]
Okay, last thing to do is wrap up one nice commit. This is a bit dangerous, but I have a backup of your patches just in case :)

# git reset a7472d927a1
# git commit -a
# git push -f

**Comment by fichtner** [comment]
(Note commit -a will pull in all of your local changes, not sure if you want that.)

**Comment by thelinuxfr** [comment]
Done :)

**Comment by fichtner** [comment]
Très chouette, merci :)

**Pull request merged**